person.rb

```diff
@@ -1,7 +1,7 @@
 require './nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] The same here, A person has many rentals so, `Getters and Setters`  for @rentals should be present as well. Please add this 👍🏻